applications/CompressiblePotentialFlowApplication/python_scripts/apply_far_field_process.py

```diff
@@ -37,7 +37,10 @@ def Execute(self):
             cond.SetValue(KratosMultiphysics.VELOCITY, self.velocity_infinity)
 
         #find the node with the minimal x
-        node1 = self.model_part.Nodes[1]
+        for node in self.model_part.Nodes:
```

Comment:
you are right, but it is not intend to be. This line is just intended to select any node belonging to the far field bcs. The node with minimal x is searched in the next lines.

Stepan was getting an error because in his mdpa the ids do not start in 1, but in a big number and therefore the line:

node1 = self.model_part.Nodes[1]

could not find the node. For this reason I substituted the line with this for loop that works for any 
order of ids. Probably there is a better solution...